src/components/shared/pick-track/index.tsx

```diff
@@ -145,7 +145,6 @@ class PickTrack extends React.Component<IPickTrackProps, IPickTrackState> {
       return <p className="pick-track__description">
           There are three batches per year, usually starting in January, May, and September.<br />
           <b>Currently, you can apply for the following batches:</b> <br />
```

Comment:
I think we should rephrase this, as there are more dates on the website, now.

Reply:
Changed it to 
"Dates and deadlines of the upcoming batches:"